components/authentication/org.wso2.carbon.analytics.idp.client/src/main/java/org/wso2/carbon/analytics/idp/client/core/utils/config/UserManagerElement.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.analytics.idp.client.core.utils.config;
+
+import org.wso2.carbon.config.annotation.Configuration;
+import org.wso2.carbon.config.annotation.Element;
+
+/**
+ * User manager Element.
+ */
+@Configuration(description = "User Manager Element")
+public class UserManagerElement {
+
+    @Element(description = "Admin Role - Display name of the role defined in the user store", required = true)
+    private String adminRole = "admin";
+
+    @Element(description = "User Store")
+    private UserStoreElement userStore = new UserStoreElement();
+
+    public String getAdminRole() {
```

Comment:
Shall we Javadoc comments to these methods?

Reply:
Since these are getters and setters used in the configuration bean, do we need doc comments for methods as well? Description of the config is given in the @Element annotation.